win32ss/user/ntuser/msgqueue.c

```diff
@@ -1946,6 +1946,7 @@ co_MsqPeekHardwareMessage(IN PTHREADINFO pti,
    DWORD QS_Flags;
    LONG_PTR ExtraInfo;
    BOOL Ret = FALSE;
+   MSG clk_msg;
```

Comment:
```suggestion
   MSG clk_msg;
   BOOL Ret = FALSE;
```
Grouping the variables that save temporary state together.

Reply:
Changed. Thanks.